test/test_seed_schema.py

```diff
@@ -64,3 +64,21 @@ def test_keeps_unrecognized_properties(self):
                            'properties': {'a': {'type': 'boolean'}},
                            'patternProperties': {r'^\d$': {'type': 'integer'}},
                            'required': ['a']})
+
```

Comment:
I don't know why this didn't occur to me before, but there should be some test that adds the same item multiple times to make sure it gets deduped.